frontends/verilog/preproc.cc

```diff
@@ -334,6 +335,11 @@ define_map_t::add(const std::string &name, const std::string &txt, const arg_map
 	defines[name] = std::unique_ptr<define_body_t>(new define_body_t(txt, args));
 }
 
+void define_map_t::add(const std::string &name, const define_body_t &body)
+{
+	defines[name] = std::unique_ptr<define_body_t>(new define_body_t(body));
```

Comment:
this looks like a job for `make_unique`

Reply:
I was thinking the same thing, but unfortunately, `make_unique` was introduced in C++14.

Reply:
... okay, nevermind then